Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.iRate</id>
+    <title>iRate for Xamarin.iOS</title>
+    <version>1.11.7</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Asks users to rate your app after they've used it for a while.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=864975</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=865084</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/unified/MTiRate.dll" target="lib/Xamarin.iOS" />
```

Comment:
Not sure that the output file will be in the unified folder, as the cake file does include unified in the build or outputs path.

Also build.cake doesn't reference include nuget as part of the build spec